admin/server/server.go

```diff
@@ -35,7 +35,7 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-const cliVersionConstraint = ">= 0.20.0"
+const minCliVersion = "0.20.0"
```

Comment:
Try `var minCliVersion = version.Must(version.NewVersion("0.20.0"))` here, then we don't have to parse it on every network request